README.md

```diff
@@ -46,17 +47,21 @@ Please note, that this is a **Partner Supported** module, which means that techn
 ### Compatibility - supported sensu versions
```

Comment:
Sensu is a proper noun as name of their product and should be capitalized. Similar for Puppet, Redis and RabbitMQ if we are referring to the tool and not a command.

Reply:
Fixed